test/jasmine/tests/choropleth_test.js

```diff
@@ -193,6 +186,18 @@ describe('Test choropleth hover:', function() {
                 d3Select('g.hovertext'),
                 style
             );
+        }
+
+        return Plotly.newPlot(gd, fig)
+        .then(function() {
+            if(hasCssTransform) {
+                transformPlot(gd, 'translate3d(10px, 10px, 0) scale(1)');
+                assertHoverLabel(pos[0] + 10, pos[1] + 10);
+                transformPlot(gd, 'translate(-25%, -25%) scale(0.5)');
+                assertHoverLabel(0.5 * pos[0], 0.5 * pos[1]);
+            } else {
+                assertHoverLabel(pos[0], pos[1]);
+            }
```

Comment:
With the latest fix, could we possibly revert changes to this test file now?

Reply:
The test changes in general are a nice to have.
However, they are not related to the latest changes. Instead, they increase the coverage by also testing the `translate3d` transform, which might behave differently from the `translate` transform but should still be usable.

Do you have a specific concern against extending the test suite in this regard?

Reply:
Similar tests using `translate` CSS are performed for other trace types.
If you want to increase coverage to take into account `translate3d` that's fine. 
So I think if possible we should keep the previous test as is and add a new one.

Reply:
okay, so the goal would be to leave the wrapper as-is so its equal to the wrapper in other tests?

To include testing for `translate3d` than would require to duplicate all 10 test cases that use this wrapper in the following lines if I understand the test setup correctly.

Reply:
Yes. Please let's revert all the changes to this file: `test/jasmine/tests/choropleth_test.js`.